### PR TITLE
[STAN-547] AND filters

### PR DESCRIPTION
**Pull request by joefitter**
* Updated all filters to be AND
* Updated type to use Select component
* Ajaxify data refreshes
* Added filterSummary component

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -29,21 +29,31 @@ jobs:
...
-      - run: npm run build --if-present
-      - run: npm start &
-      - run: npm run test:integration
+          working-directory: ui
+          build: npm run build --if-present
+          start: npm start
+        env:
+          CKAN_URL: https://manage.test.standards.nhs.uk/api/action
+          PORT: 3000
+          CYPRESS_host: test.standards.nhs.uk
```
Comment (roc) [inline .github/workflows/build-and-test.yml:47]: probably not necessary

Comment (lennym) [reply .github/workflows/build-and-test.yml]: What do you _think_ this is doing?

Comment (roc) [reply .github/workflows/build-and-test.yml]: removed

File: ui/cypress/integration/standards/list.js [hunk 2]
```diff
@@ -46,9 +49,15 @@ describe('Standards Listing Index', () => {
...
         cy.get('input[name="q"]').type('prsb');
-
         cy.contains('Search').click();
-        cy.get('#browse-results li a').eq(0).click();
+
+        cy.get('#resultSummary')
+          .invoke('attr', 'data-loading')
+          .should('eq', 'false');
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(500);
```
Comment (lennym) [inline ui/cypress/integration/standards/list.js:58]: Is this still required?

File: ui/components/Dataset/index.js [hunk 3]
```diff
@@ -148,35 +139,72 @@ const NoResultsSummary = ({ searchTerm }) => (
...
+      sort,
+      filters,
+    };
+
+    try {
+      setLoading(true);
+      const res = await axios.post('/api/refresh-list', params);
+      setData(res.data);
+    } catch (err) {
+      console.error(err);
```
[the remark this answers is not in the document]
Comment (easternbloc) [reply ui/components/Dataset/index.js:188]: Agreed!

[the remark this answers is not in the document]
Comment (roc) [reply ui/components/Dataset/index.js:189]: done 👍

File: ui/cypress/integration/standards/list.js (hunk 2 above)
Comment (roc) [reply ui/cypress/integration/standards/list.js:58]: Sadly yes, I'm not sure _why_ but I'm also sure I'm at the end of being able to care about cypress edge cases on dom waits :(

**Comment by easternbloc** [comment]
I'm gonna merge this in. Not loving the wait but I walked through it with ralph and it appears to be needed.

**Pull request merged**